include/common_structs.h

```diff
@@ -1187,7 +1187,19 @@ typedef struct CollisionStatus {
 } CollisionStatus; // size = 0x28
 
 typedef struct DecorationTable {
-    /* 0x000 */ char unk_00[2011];
+    /* 0x000 */ char unk_00[1729];
```

Comment:
If an unk is being used by code, could you make the `char`s s8 instead? `char` in structs is used to denote unknown space